tests/integration/cache_test.go

```diff
@@ -531,28 +532,40 @@ func TestGet(t *testing.T) {
 func testGet(t *testing.T, kv clientv3.KV, getReader func() Getter, initialEvents, followupEvents []*clientv3.Event) {
 	ctx := t.Context()
 	t.Log("Setup")
-	lastRev := applyEvents(ctx, t, kv, initialEvents)
+	setupCompleteRev := applyEvents(ctx, t, kv, initialEvents)
```

Comment:
The "Setup" step is from `t.Log("Setup")` on line 534 to `t.Log("Validate")` line 551. The name here totally doesn't make sense.

Best would be to separate `initialEvents` with `initialRev` and `followupEvents` with `followupRV`